client/ngapp/build/main.js

```diff
@@ -66,6 +65,15 @@ angular.module('ooniAPIApp')
         console.log('error', error)
       })
 
+    Country.findOne({
```

Comment:
I'm getting `Country is not defined` here, probably needs to be imported in the controller?